Checker_board.py

```diff
@@ -0,0 +1,171 @@
+w, h = 8, 8;
```

Comment:
is this checker_board.py needed for the actual pr or is this still code your working on

Reply:
No, checker_board.py im not working on that. May be i would need to delete it from here